packages/safe-core-sdk/README.md

```diff
@@ -108,7 +108,7 @@ Before executing this transaction, it must be signed by the owners and this can
 The `owner1` account signs the transaction off-chain.
 
 ```js
-const owner1Signature = await safeSdk.signTransaction(safeTransaction)
+const signedSafeTransaction = await safeSdk.signTransaction(safeTransaction)
 ```
 
 Because the signature is off-chain, there is no interaction with the contract and the signature becomes available at `safeTransaction.signatures`.
```

Comment:
Should we rephrase this as well to include `signedSafeTransaction`?

Reply:
True. Good catch, thanks!